### ISSUE TITLE
Updates on IOS 17.4?

### ISSUE THREAD

**Issue by Tazi0** [altstoreio/AltStore#1400]
Since Apple has allowed 3rd party app stores in the EU are there any plans to easily install AltStore onto your iPhone?

**Comment by acmpo6ou** [comment]
Here is the post they made on Mastodon a while back: https://fosstodon.org/@altstore/111818192849973994

But yeah, I'd like to know how that's going, and which struggles they have with it. Like, will they have to pay for apps that have more than 1M downloads monthly?

**Comment by Tazi0** [comment]
Pushed back to April:
https://www.threads.net/@altstoreio/post/C5No6nXOC0I/

**Comment by acmpo6ou** [comment]
Yes I saw a post on Mastodon as well a few days ago. Nice you posted it here.

**Comment by alextarana** [comment]
Can we have updates on this?

**Comment by Tazi0** [comment]
https://www.threads.net/@altstoreio/post/C5rKCAOueF2/

Its currently "In Review" at apple

**Comment by acmpo6ou** [comment]
Well there we go, AltStore was approved by Apple, gentlemen and ladys! https://fosstodon.org/@altstore/112287538562764439

**Comment by ssaavedra** [comment]
Will iPad be supported? Currently if we try to follow the link on an iPad a system message says "this app store only works on iPhone".

**Comment by Tazi0** [comment]
Does anyone know of a different payment method will be supported like paypal? credit card isnt a option for everybody

**Comment by acmpo6ou** [comment]
> Does anyone know of a different payment method will be supported like paypal? credit card isnt a option for everybody

Exactly, I agree, I think you can contact them for that by email? Or make an issue on GitHub

**Comment by Tazi0** [comment]
> > Does anyone know of a different payment method will be supported like paypal? credit card isnt a option for everybody
> 
> Exactly, I agree, I think you can contact them for that by email? Or make an issue on GitHub

@acmpo6ou I've emailed them and will reply with their response once received.
